x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_form.test.tsx

```diff
@@ -608,6 +609,7 @@ describe('rule_form', () => {
     it('renders rule name', async () => {
       await setup();
       const ruleNameField = wrapper.find('[data-test-subj="ruleNameInput"]');
+
```

Comment:
same here? just newlines were added

Reply:
Yes, it's a leftover after backing out changes of inputs optimisation try in alert flyout.